client/blocks/login/login-form.jsx

```diff
@@ -292,7 +292,7 @@ export class LoginForm extends Component {
 										: this.props.translate( 'Change Username' ) }
 								</a>
 							) : (
-								this.props.translate( 'Email Address or Username' )
```

Comment:
Why not mention username here?
I think it's better to have the previous copy here

Reply:
@yurynix: see the whole discussion above ;)

Reply:
Maybe I didn't phrased that right, I _think_ we should have the username here as it used to be.